src/tools.cpp

```diff
@@ -346,6 +346,15 @@ void trimString(std::string& str)
 	str.erase(0, str.find_first_not_of(' '));
 }
 
+bool hasSameSign(int32_t x, int32_t y, bool neutralZero /* = true */)
+{
+	if(neutralZero) {
```

Comment:
Well, perhaps it's redundant at the moment, but I'm afraid someone may use it in the future with neutralZero set to false and it will give unexpected behaviour.

Thank you for your suggestions, I've made changes based on them, apart from this one. I think it would be better to leave it to avoid confusion when used in the future.